src/main/java/org/embulk/output/dynamodb/DynamodbUtils.java

```diff
@@ -178,11 +203,22 @@ protected String getPrimaryKeyName(DynamoDB dynamoDB, String tableName)
         String primaryKey = null;
         while (schema.hasNext()) {
             KeySchemaElement element = schema.next();
-            primaryKey = element.getAttributeName();
+            if (element.getKeyType().equals(KeyType.HASH.toString())) {
+                primaryKey = element.getAttributeName();
+            }
         }
         return primaryKey;
     }
 
+    protected List getPrimaryKey(DynamoDB dynamoDB, String tableName)
+    {
+        log.debug("getPrimaryKey() called");
```

Comment:
I missed it. I also think that it is not necessary.
I deleted unnecessary log output and committed it.